pydis_site/apps/content/resources/guides/python-guides/fix-ssl-certificate.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Fixing an SSL Certificate Error.
+description: A guide on fixing an SSL certificate error.
+---
+#### How to fix SSL Certificate issue on Windows
+
+Firstly, try updating your OS, wouldn't hurt to try.
+Now, if you're still having an issue, you would need to download the certificate for the SSL.
+The SSL Certificate, Sectigo (cert vendor) provides a download link of [certificate](https://crt.sh/?id=2835394)
```

Comment:
This is missing a trailing period. You can put multiple sentences on one line, although split them in the middle to make sure they don't get too long.